hr_phone_validation/models/hr_employee.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models
+
+
+class HrEmployee(models.Model):
+    _name = 'hr.employee'
+    _inherit = ['hr.employee', 'phone.validation.mixin']
+
+    @api.onchange('work_phone', 'country_id')
+    def _onchange_work_phone_validation(self):
+        if self.work_phone:
+            self.work_phone = self.phone_format(self.work_phone)
```

Comment:
Hi @fmdl , small module but interesting, i have a remark that here you should add the employee country as parameter, since it can be different from the company country...

Reply:
HI @feketemihai you don't need to add country=self.country it is already built in here : https://github.com/odoo/odoo/blob/11.0/addons/phone_validation/models/phone_validation_mixin.py#L11

Reply:
You're right, thanks for the explanation...